src/transactions/multisignature.js

```diff
@@ -66,7 +66,7 @@ function createTransaction(
  * @return {Object}
  */
 
-function createMultisignature(secret, secondSecret, keysgroup, lifetime, min, timeOffset) {
+export function createMultisignature(secret, secondSecret, keysgroup, lifetime, min, timeOffset) {
```

Comment:
Does the fact that this breaks the pattern suggest these functions should be in different files?